app/scripts/modules/core/delivery/executionGroup/execution/execution.directive.js

```diff
@@ -94,6 +94,15 @@ module.exports = angular
       });
     };
 
+    this.restartDetails = () => {
```

Comment:
Agreed - no need to make it a function. You could unwrap it and just set a value on `this`, i.e.
```javascript
      let restartedStage = this.execution.stages.find(stage => stage.context.restartDetails);
      if (restartedStage) {
        this.restartDetails = restartedStage.context.restartDetails;
      }
```
then, in your html template, just drop the `()`